Part 2

```diff
@@ -0,0 +1,14 @@
+Matthew Danielson
+6/29/16
+Assignment 2; Part 2
+
+1) Yes, multiple operators can appear on each line, for instance: if(a == b || b == c) : there can be multiple operators used to further specify the restrictions on a conditional.
+ 
+2) Logical errors are mistakes by a programmer where there the code is not faulty due to a missing semi colon or quotation mark - which are syntax errors - but rather, when the rational on a conditonal or loop is incorrect.
```

Comment:
yup! It's important to also note that logical errors occur during runtime and syntax errors occur during compile time.